tomcat-main/src/main/resources/defaults/base-procedure-dutyofcare.xml

```diff
@@ -0,0 +1,38 @@
+<record id="dutyofcare" in-findedit="yes" type="record,procedure" cms-type="default" generate-services-schema="true" >
+  <services-url>dutyofcares</services-url>
+  <services-tenant-plural>Dutyofcares</services-tenant-plural>
+  <services-tenant-singular>Dutyofcare</services-tenant-singular>
+  <services-list-path>abstract-common-list/list-item</services-list-path>
+  <services-record-path>dutyofcares_common:http://collectionspace.org/services/dutyofcare,dutyofcares_common</services-record-path>
+  <services-record-path id="collectionspace_core">collectionspace_core:http://collectionspace.org/collectionspace_core/,collectionspace_core</services-record-path>
+
+  <include src="domain-procedure-dutyofcare.xml" strip-root="yes" />
+
+  <section id="coreInformation">
+    <include src="core-fields.xml" strip-root="yes" />
+  </section>
+
+  <section id="dutyOfCareInformation">
+    <field id="dutyOfCareNumber" mini="number,list" />
+    <field id="originationDate" datatype="date" />
+    <field id="dutyOfCareTitle" mini="summary,list" />
+    <repeat id="dutyOfCareNotes">
+      <field id="dutyOfCareNote" />
+    </repeat>
+
+    <repeat id="partiesInvolvedGroupList/partiesInvolvedGroup">
+      <field id="involvedParty" />
+      <field id="involvedOnBehalfOf" />
+      <field id="involvedRole" />
+    </repeat>
+
+    <repeat id="dutyOfCareDetailsGroupList/dutyOfCareDetailsGroup">
+      <field id="detailType" />
+      <field id="detailLevel" />
+      <field id="detailDeterminedBy" />
```

Comment:
This is an autocomplete in the UI, so should have `autocomplete="true"` (the services layer needs to know which fields can contain ref names).